app/api/routes/product.py

```diff
@@ -0,0 +1,154 @@
+from fastapi import APIRouter, HTTPException, Depends,Query
+from uuid import UUID, uuid4
+from typing import Dict, Optional
+from pydantic import BaseModel, Field
+from dependencies import get_current_time,get_current_admin,products_db
+from models import Product,ProductCreate,ProductUpdate
+
+
+
+router = APIRouter()
+    
+"""Create a Product"""
+@router.post("/products/", status_code=201)
+async def create_product(product: ProductCreate, admin: bool = Depends(get_current_admin)):
+    if not admin:
```

Comment:
This one should be implemented in the get_current_admin function

Reply:
Done